tutorials/MD/03_MD_Setup_Production_Run.ipynb

```diff
@@ -0,0 +1,274 @@
+    "\n",
+    "- Coordinates files have the extension .gro and the default name is conf.gro.\n",
+    "- The topology file (default name topol.top) contains all the information about which atoms are bonded to which and what force-field parameters are applied etc.\n",
+    "- The trajectory files have the extension .xtc and .trr, the former does not contain velocity information and coordinates are held at a reduced precision, and so occupies less disk space. However you will need velocities if you want to continue a simulation.\n",
+    "- The .edr file contains the energy information from the trajectory.\n",
+    "- The .mdp file contains the information that was used to setup the actual simulation. Things related to temperature, pressure, how the electrostatics is calculated etc. Although we will not use this today, the file is provided so you can see how this trajectory was made.\n",
+    "- The .ndx file allows you to specify atoms or groups of atoms for use in analysis, restraints, etc and is optional.\n",
+    "- The .tpr file is a binary file that contains all the information needed to perform the actual run (this allows  gromacs to do lots of self-consistency checks to minimize user errors)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Molecular Dynamics Simulation of Protein\n",
+    "\n",
+    "### Protein System Setup\n",
+    "\n",
+    "In this section, we will obtain our protein coordinates and perform some routine Molecular Dynamics calculations on them.\n",
+    "\n",
+    "For this tutorial, we will use the HIV-1 protease structure (1HSG). It is a homodimer with two chains of 99 residues each. The .pdb file of the protein can be obtained from the protein data bank (https://www.rcsb.org/), but it is probably easier for now to just copy it from the shared directory to your working directory. In fact, most of the files you need can be found in this directory so if you get confused/stuck/lost etc then you can always look here to check you did things correctly.\n",
+    "\n",
+    "`% cp /home/shared/data/1hsg.pdb .` (path to be verified)\n",
+    "\n",
+    "If you look at this file (using the graphical editors vi or nedit for example, you should immediately see that it has two chains; A and B:\n",
+    "\n",
+    "`% vi 1hsg.pdb`\\\n",
+    "or\\\n",
+    "`% nedit 1hsg.pdb`\n",
+    "\n",
+    "Now we need to prepare our protein for simulation. First of all we will extract only the protein coordinates from the pdb file into a new file called `protein.pdb`. To do this enter the following command:\n",
+    "\n",
+    "`% grep ATOM 1hsg.pdb > protein.pdb`\n",
+    "\n",
+    "Use your preferred text editor to open the protein.pdb file and see how it differs from the 1hsg.pdb file.\n",
+    "\n",
+    "We now need to make sure that all the hydrogens are added to our protein. This process will also generate the\n",
```

Comment:
It might be more easy to understand if this sentence is reversed, i.e. generate the parameters, and will also add hydrogens where necessary (should we also detail what -ignh does)?

Reply:
Done